### PR TITLE
Hotfix/navigation

### PR DESCRIPTION
**Pull request by matthijs186**

### CODE REVIEW

**Review by ferrybig — APPROVED** [review]

**Review by Drelofs — APPROVED** [review]
Good! Merge!

**Review by RishiTheGoat — APPROVED** [review]
👍

File: src/assets/scss/partials/_nav.scss [hunk 1]
```diff
@@ -58,15 +60,20 @@ nav.main-nav {
 
 	> .main-nav-list > li:hover {
 		background-color: mix($primary-green, $white, 60%);
+		padding-left: 5px;
+		text-align: left;
```
Comment (peterxqe) [inline src/assets/scss/partials/_nav.scss]: same as not hover

**Review by peterxqe — CHANGES_REQUESTED** [review]
👌

**Review by peterxqe — APPROVED** [review]
ok..

**Pull request merged**